apps/site/lib/site_web/controllers/helpers.ex

```diff
@@ -178,4 +178,8 @@ defmodule SiteWeb.ControllerHelpers do
 
     "#{Strftime.format!(one_year_after, "%d %b %Y")} 00:00:00 EST"
   end
+
+  def call_plug_with_opts(conn, module, opts) do
```

Comment:
I wonder if adding a test for this would help with coverage?

Reply:
Still no luck 😞 
Any chance we can merge this regardless and perhaps spin up a new PR with the additional tests for Foxboro?